lms/djangoapps/commerce/views.py

```diff
@@ -111,7 +111,8 @@ def post(self, request, *args, **kwargs):  # pylint: disable=unused-argument
 
         try:
             timeout = getattr(settings, 'ECOMMERCE_API_TIMEOUT', 5)
-            response = requests.post(url, data={'sku': course_modes[0].sku}, headers=headers, timeout=timeout)
+            response = requests.post(url, data=json.dumps({'sku': course_modes[0].sku}), headers=headers,
```

Comment:
It's fine for now, since this is going to have to change in the future as we handle verified, professional, etc. It just stood out to me.